lib/session.js

```diff
@@ -378,7 +378,7 @@ function longSms(pduObj) {
 	                                                                              // 4th octet and length is
 	                                                                              // header size -2 octets
 	    partsCount    = pduObj.params.short_message[2 + csmsReference.length + 1],
-	    partNr        = pduObj.params.short_message[2 + csmsReference.length + 1],
+	    partNr        = pduObj.params.short_message[2 + csmsReference.length + 2],
```

Comment:
This will only work for UDH headers that are 2 in size, right?

Don't we need to use the UDH size instead of hardcoded 2 here?